tests/Query/Mysql/DateTest.php

```diff
@@ -36,6 +36,15 @@ public function testDateAdd2()
         $this->assertEquals($sql, $q->getSql());
     }
 
+    public function testFromUnixtime()
+    {
+        $dql = "SELECT p FROM DoctrineExtensions\Tests\Entities\Date p WHERE FROM_UNIXTIME(CURRENT_TIME())";
+        $q = $this->entityManager->createQuery($dql);
+
+       $sql = "SELECT d0_.id AS id0, d0_.created AS created1 FROM Date d0_ WHERE FROM_UNIXTIME(CURRENT_TIME())";
+       $this->assertEquals($sql, $q->getSql());
```

Comment:
Ah, this fails because your `WHERE` statement is invalid, because you're not comparing it to anything, you need something like:

```
FROM_UNIXTIME(CURRENT_TIME()) = '2000-01-01 00:00:00'
```

on both, the DQL and SQL lines, then it should pass